Terraform-tfvars.sublime-syntax

```diff
@@ -0,0 +1,28 @@
+%YAML 1.2
+#
+# This syntax definition is based on the Terraform guide:
+# https://www.terraform.io/docs/configuration/index.html
+#
+# As well as the HCL Native Syntax Spec:
+# https://github.com/hashicorp/hcl2/blob/master/hcl/hclsyntax/spec.md
+#
+# For documentation on the .subline-syntax format:
+# https://www.sublimetext.com/docs/syntax.html
+#
+# Regex's in this file support the Oniguruma regex engine:
+# https://raw.githubusercontent.com/kkos/oniguruma/5.9.6/doc/RE
+#
+---
+name: Terraform (vars)
```

Comment:
```suggestion
name: Terraform (variables)
```

Reply:
I was contemplating this and file names myself but I think that your suggestions work just fine.

Here I've opted for something shorter to take less space in the status bar but either works really.

Reply:
Oh I see, I haven't considered the UI aspect of this particular wording

Reply:
Your suggestion is fine. I don't like abbreviates either way.